src/components/common/SearchPannel/SearchPannel.jsx

```diff
@@ -0,0 +1,19 @@
+import './searchPannel.scss';
+
+const SearchPannel = () => {
+  return (
+    <form className="search-container" action="/search" method="GET">
+      <button type="submit" className="search-button">
```

Comment:
I don't think a search input works that way by clicking on the magnifier icon

Check this example
https://mui.com/material-ui/react-autocomplete/#search-input

Reply:
You might see the similar experience in LinkedIn

<img width="521" alt="image" src="https://github.com/voyageglobal/voyage-frontend/assets/19875209/14d1d6b3-bbb9-42b7-882a-53794389bcd5">